docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -0,0 +1,264 @@
+command, the configuration can be provided as an inline JSON or YAML string,
+such as in this example:
+
+[,bash]
+----
+ec validate image --policy '{
+    "configuration": {
+        "collections": ["minimal"]
+    }
+    "sources": [
+        {
+            "policy": ["oci::https://quay.io/hacbs-contract/ec-release-policy:latest"],
+            "data": ["oci::https://quay.io/hacbs-contract/ec-policy-data:latest"]
+        }
+    ]
+} ...'
+----
+
+As a reference to a xref:ecc:ROOT:reference.adoc[EnterpriseContractPolicy]
+Kubernetes custom resource by either name (e.g. "my-policy") in the current
+namespace, or by providing namespace and the name separaded by slash (`/`)
+character, such as in this example:
+
+[,bash]
+----
+ec validate image --policy my-namespace/my-policy
+----
+
+If the explicit policy is not provided via the `--policy` parameter, the default
+policy named `default` is loaded from `enterprise-contract-service` namespace of
+the cluster accessed using the current Kubernetes client configuration.
+
+== Including and excluding rules
+
+By default, all rules are included.
+
+You can modify the defaults by setting `config.policy.include` and
+`config.policy.exclude`. Each of those values should be a list of strings.
+
+The strings in the list should be one of the the following:
+
+A "package name"::
+
+Package names can be found in the policy documentation, for example the
+xref:ec-policies:ROOT:release_policy.adoc#attestation_type_package[Attestation Type] package
+name is `attestation_type`. Specifing a package name by itself in the include
+or exclude list means every rule from that package should be included or
+excluded.
+
+A "rule name"::
+
+A rule name consists of the rule's package name and the rule's "code" separated
+by a dot. Rule codes can be see in the documentation also. For example the
+xref:ec-policies:ROOT:release_policy.adoc#attestation_type__unknown_att_type[Unknown attestation type found] rule
+name is `attestation_type.unknown_att_type`.
+
+A "package name:term"::
+
+Some policy rules process a list of items, "term" allows a particular item to
+be excluded or included. This matcher behaves like "package name" but only
+applies to policy rules in the package that match the "term". For example, the
+xref:ec-policies:ROOT:release_policy.adoc#test_package[Test package] emits results for each
+test case. A particular test case can be ignored without ignoring the remaining
+ones.
+
+A "rule name:term"::
+
+This is similar to "package name:term", but allows finer granularity by only
+applying to a particular policy rule within the package.
+
+"*"::
+
+The `"\*"` acts as a wildcard to match any package. Note that it does not match
+partial names, so you can't specify `"s*"` to match every package starting with
+"s" for example.
+
+You can also specify `"<packagename>.*"` and it works the same as just
+`"<packagename>"` to represent every rule in a package.
+
+== Specifying that individual tests are non-blocking
```

Comment:
I think this can be omitted now since it's covered by the `:term` options above. `test` is no longer a special case. The "Specifying that certain tests should be non-blocking" below is still relevant and highlights this.